cmd/ndm_daemonset/app/command/commands.go

```diff
@@ -46,6 +46,9 @@ func NewNodeDiskManager() (*cobra.Command, error) {
 	cmd.PersistentFlags().StringVar(&options.ConfigFilePath, "config",
 		controller.DefaultConfigFilePath,
 		"Path to config file")
+	cmd.PersistentFlags().StringSliceVar(&options.FeatureGate, "feature-gates",
+		nil,
+		"FeatureGates to be enabled or disabled")
```

Comment:
usually only enable right?

Reply:
Some features can also be kept disabled. In case of slowly deprecating a feature, we keep it disabled and users can see how it will affect their system.